app/Providers/AppServiceProvider.php

```diff
@@ -12,6 +12,26 @@ class AppServiceProvider extends ServiceProvider
      * @return void
      */
     public function register()
+    {
+        $this->app->configure('cdn');
+        $this->app->configure('media');
+        $this->app->configure('ratelimiter');
+        $this->app->configure('multisite');
+        $this->app->configure('ohanzee-db');
+
+        $this->registerServicesFromAura();
```

Comment:
why does it matter that it is using Aura to register usecase and repository.message?